fire-hose-management-frontend/src/components/Departments/EditDepartmentOverlay.jsx

```diff
@@ -0,0 +1,186 @@
+    }
+
+
+    console.log(department)
+
+    return (<Overlay
+        onClose={() => handleClose()}
+        headerContent={<h2>{department.departmentName} aktualisieren</h2>}
+        size={"m"}>
+        <ValidatorForm
+            onSubmit={() => editDepartmentMutation.mutate(departmentBuilder(gender, firstname, lastname, mail, sendConfirmationMail))}>
+            <Grid container spacing={3}>
+                <Grid item xs={12}>
+                    <TextField
+                        label={"Abteilung"}
+                        value={department.departmentName}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <TextField
+                        label={"Straße"}
+                        value={department.street}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <TextField
+                        label={"Hausnummer"}
+                        value={department.houseNumber}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <TextField
+                        label={"PLZ"}
+                        value={department.postalCode}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <TextField
+                        label={"Ort"}
+                        value={department.location}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <Select
+                        label={"Land"}
+                        value={"Deutschland"}
+                        variant="filled"
+                        fullWidth={true}
+                        disabled={true}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CSelect
+                        label={"Anrede"}
+                        defaultValue={[]}
+                        value={gender}
+                        onChange={(event) => setGender(event.target.value)}
+                        options={genderMenuItems}
+                        validators={["required"]}
+                        errorMessages={["Die Auswahl einer Anrede ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CTextField
```

Comment:
Codacy has a fix for the issue: [Delete `········`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
            <CTextField
```